### PR TITLE
Some typos fixed

### PR DESCRIPTION
**Pull request by vlajos**
Some typos fixed.
Found by https://github.com/vlajos/misspell-fixer

(I have reviewed it manually and reverted some false positives, suspicious ones, but I am not too familiar with the codebase...)

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -925,7 +925,7 @@
       o Overhauled Win32 builds
       o Cleanups and fixes to the Big Number (BN) library
       o Support for ASN.1 GeneralizedTime
-      o Splitted ASN.1 SETs from SEQUENCEs
+      o Split ASN.1 SETs from SEQUENCEs
```
Comment (mspncp) [inline NEWS]: The NEWS items should remain as they are. Please revert this change.

File: crypto/property/property_err.c [hunk 2]
```diff
@@ -25,7 +25,7 @@ static const ERR_STRING_DATA PROP_str_reasons[] = {
     {ERR_PACK(ERR_LIB_PROP, 0, PROP_R_NOT_A_DECIMAL_DIGIT),
     "not a decimal digit"},
     {ERR_PACK(ERR_LIB_PROP, 0, PROP_R_NO_MATCHING_STRING_DELIMETER),
-    "no matching string delimeter"},
+    "no matching string delimiter"},
```
Comment (mspncp) [inline crypto/property/property_err.c]: This file is autogenerated, the string text is generated from the symbol, `PROP_R_NO_MATCHING_STRING_DELIMETER`, which already contains the typo.

If I'm not mistaken, the constant can be renamed, because it was introduced only recently in the master branch.

**Comment by mspncp** [comment]
The approval holds, provided the typo in the symbol is fixed, too:

https://github.com/openssl/openssl/blob/14ee781eef0e55563432f377d8911529823bee58/crypto/err/openssl.txt#L2678
https://github.com/openssl/openssl/blob/14ee781eef0e55563432f377d8911529823bee58/crypto/property/property_err.c#L27
https://github.com/openssl/openssl/blob/14ee781eef0e55563432f377d8911529823bee58/crypto/property/property_parse.c#L206
https://github.com/openssl/openssl/blob/14ee781eef0e55563432f377d8911529823bee58/include/internal/propertyerr.h#L46

File: NEWS (hunk 1 above)
Comment (vlajos) [reply NEWS]: I have reverted the NEWS file's change.

File: crypto/property/property_err.c (hunk 2 above)
Comment (vlajos) [reply crypto/property/property_err.c]: I have applied the fix for the constant as well.

**Comment by vlajos** [comment]
The changes:
https://github.com/openssl/openssl/compare/7a74d43f895002974865270f0c858c305434008d..085c4dee64d18ce541e17659db68738680bb9cd7
(And the commit message.)

**Comment by mspncp** [comment]
> The changes:
> https://github.com/openssl/openssl/compare/7a74d43f895002974865270f0c858c305434008d..085c4dee64d18ce541e17659db68738680bb9cd7
> (And the commit message.)

That was really quick. And thanks for providing the link, but actually GitHub already did this for you: if you look a few lines up, you will see that the **force-pushed** is actually a link to the force-push diff.

image

So as long as you don't move the merge-base, force-pushing is ok. (Otherwise, it is frowned upon during review ;-) ).



One last thing:  I noticed that it was stupid of me to add your name to the `Reported-by:` line, since you are already the author. (I did it by habit, because I normally add the reporter's name if he raised an issue, not a pull request.)

So if you like, you could change the line to

```
Reported-by: misspell-fixer <https://github.com/vlajos/misspell-fixer>
```

but you don't have to do it.

**Comment by levitte** [comment]
Side note: had to look up "publicly" vs "publically". There's an interesting discussion about that on english.stackexchange.com...

**Comment by vlajos** [comment]
> Side note: had to look up "publicly" vs "publically". There's an interesting discussion about that on english.stackexchange.com...

I do not really have a strong opinion about this one.
If you prefer to keep 'publically' I am happy to revert these bits.
It seems that it is not really a clear-cut.

**Comment by levitte** [comment]
I'm not sure what gave off a message that I dislike the change to "publicly", so let me assure you it's fine.  I did write "Side note" for a reason :wink:

**Comment by levitte** [comment]
Merged.

master:
79c44b4e3044aee9dc9618850d4f1ce067757b4b Fix some typos

1.1.1:
3986b9bb6bb319ff4b2365a7d570308d0e70cd54 Fix some typos
